### DISCUSSION TITLE
How to make translation json file keys order match the source file

### DISCUSSION THREAD

**Discussion by mokazemi** [WeblateOrg/weblate#9421] [Q&A]
Hi. I'm using Weblate to translate Jitsi Meet into my language (Farsi). 
By the way, Weblate adds new translation at the end of the translation json file so the order of keys in the translation and the main source file is different. How can I make it match with the source file?

**Comment by nijel** [comment]
You can sort the file using https://docs.weblate.org/en/latest/admin/addons.html#customize-json-output

**Reply by mokazemi** [reply]
I enabled it but I don't see any option for it in the settings. There's also nothing explained in the docs.

**Reply by nijel** [reply]
All options are configured at add-on installation, you can go back to that via *Manage* / *Add-ons* on the component.
